docs/design/pnc.md

```diff
@@ -0,0 +1,113 @@
+**# Design document for PNC package manager
+
+Contents:
+
+1. [PNC](#PNC)
+2. [Current implementation overview (OSBS)](#overview-of-the-current-implementation-in-osbs)
+3. [Design for the Cachi2 implementation](#design-for-the-implementation-in-cachi2)
+
+## PNC
+
+### Glossary
+- **PNC**: Project NewCastle - [open-source project](https://github.com/project-ncl/pnc) for managing, executing, 
+  and tracking cross-platform builds.
+
+
+### Repository configuration 
```

Comment:
This part seems to be OSBS specific, so it is probably better fit into the `Overview of the current implementation in OSBS` section.

Reply:
Moved